manifests/04_configmap.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-insights
+  name: insights-operator-config
+data:
+  operator-config.yaml: |
```

Comment:
This breaks CI.  Why did you remove the default config?

Reply:
You shouldn't override this, instead the config map is a customization.  Our default ootb is set via config/pod.yaml so that way you don't have to do this.

Reply:
I've mainly inspired by https://github.com/openshift/cluster-authentication-operator.

I agree with config maps to serve mainly for overrides. Is there some good operator that would be following this pattern of having static file as defautls + config file from configmaps for overrides?